owncloudApp/src/main/java/com/owncloud/android/ui/activity/UploadPathActivity.java

```diff
@@ -70,7 +71,7 @@ protected void onAccountSet(boolean stateWasRecovered) {
             }
 
             if (!stateWasRecovered) {
-                MainFileListFragment listOfFolders = getListOfFilesFragment();
+                MainFileListFragment listOfFolders = getListMainFileFragment();
```

Comment:
If we are updating the name, we can update it to the proper one, right?
```suggestion
                MainFileListFragment listOfFolders = getMainFileListFragment();
```